src/backend/distributed/operations/shard_cleaner.c

```diff
@@ -490,15 +498,17 @@ FinalizeOperationNeedingCleanupOnFailure()
 
 	if (list_length(currentOperationRecordList) > 0)
 	{
-		ereport(LOG, (errmsg("Removed %d orphaned shards out of %d",
-							 removedShardCountOnComplete, list_length(
-								 currentOperationRecordList))));
+		ereport(LOG, (errmsg(
+						  "Processed %d orphaned shards out of %d as part of failed operation finalize ",
```

Comment:
since this will also be exposed to users, "finalize" might be a bit confusing since it's an implementation (naming) detail

Maybe: "removed %d orphaned shards out of %d as part of a failed operations"

Reply:
We initially had verbiage like : "Removed 34 orphaned shards out of 67"
The suggestion was to disable the logging for cleanup of resources during the operation as it can be concerning : https://github.com/citusdata/citus/issues/6312.

This is why the Processed verbiage, do we want to go back to Remove?

Reply:
Fixed as per offline discussion.